### PR TITLE
More dns changes

### PR DESCRIPTION
**Pull request by lowflying**
### JIRA link (if applicable) ###

N/A

### Change description ###

updated the names of the endpoints and subsequent A records - it was previoously not creating an A record with the correct name

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by thomaswatsonv1 — DISMISSED** [review]
Looks good!

File: MISSING_SECRETS [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+ServiceBusConnection
+kinlyconfiguration--apisecret
+kinlyconfiguration--callbacksecret
+notifyconfiguration--apikey
+testing--testuserpassword
+cvpconfiguration--managedidentityclientid
+defaultpassword 
+TestDefaultPassword 
+azuread--temporarypassword
+testusersecrets--testuserpassword
```
Comment (chris-pateman) [inline MISSING_SECRETS:12]: Do you need these added?

**Review by chris-pateman — APPROVED** [review]
LGTM. 
just need to remove secret list

**Review by chris-pateman — APPROVED** [review]
LGTM

**Pull request merged**